### ISSUE TITLE
detectnet mAP=0 with ms-coco datasets

### ISSUE THREAD

**Issue by luhongwei** [NVIDIA/DIGITS#2107]
I am faced the mAP=0 probelm. I pulled about 1000 pictures with boats from MS-COCO data set in order to train a model that recognizes boats. But on my DIGITS, after about 300 epochs, the conv1_7x7_s2 became so "pale" like in attached picture even through I had started training from a pre-trained model, detectnet(KITTI)
![conv1_7x7_s2](https://user-images.githubusercontent.com/4384919/45069047-26dc8200-b0fd-11e8-9fd8-bcb666c1082a.jpeg)

I'm wondering if the training set was not good enough since I put small boats pictures in a generated 1024x768 background. like in attached
![sourceimg](https://user-images.githubusercontent.com/4384919/45069063-3d82d900-b0fd-11e8-83a3-5164e053f2d7.jpeg)

By the way, I did use Alps Labeling Tool to check the correctness of labels.
